testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -1724,7 +1724,19 @@ def test_set_dimensions(self):
         u.dimensions = np.array([10, 11, 12, 90, 90, 90])
         assert_allclose(u.dimensions, box)
 
+    def test_universe_kwargs(self):
```

Comment:
I like QC's rationale for making it `@staticmethod`:

"This will improve performance and make it clear to users of your code that the function does not use any attributes of the class."

We should do it when we can.